src/components/views/dialogs/ConfirmUserActionDialog.js

```diff
@@ -55,7 +55,7 @@ export default createReactClass({
         askReason: false,
     }),
 
-    componentWillMount: function() {
+    componentDidMount: function() {
         this._reasonField = null;
```

Comment:
this might overwrite the ref depending on the run order between refs and did-mount

Reply:
![image](https://user-images.githubusercontent.com/2403652/78164726-1a87ea80-7442-11ea-96cd-c775233a9038.png)

Reply:
refs update before componentDidMount so this will clear the ref after first render.